Example/FinancialConnections Example/FinancialConnectionsUITests/FinancialConnectionsUITests.swift

```diff
@@ -291,7 +291,10 @@ final class FinancialConnectionsUITests: XCTestCase {
         }
         institutionButton.tap()
 
-        let prepaneContinueButton = app.webViews.buttons["Continue"]
+        let prepaneContinueButton = app.webViews
+            .buttons
+            .containing(NSPredicate(format: "label CONTAINS 'Continue'"))
```

Comment:
This makes it more robust where instead of expecting it to always be "Continue" it is now flexible to having "Continue" anywhere in the button